web/docs/tql2/operators/compress_brotli.md

```diff
@@ -0,0 +1,48 @@
+# compress_brotli
+
+Compresses a stream of bytes using Brotli compression.
+
+```tql
+compress_brotli [level=int, window_bits=int]
+```
+
+## Description
+
+The `compress_brotli` operator compresses bytes in a pipeline incrementally.
+
+:::note Streaming Compression
+The operator uses [Apache Arrow's compression
+utilities][apache-arrow-compression] under the hood, and transparently supports
+all options that Apache Arrow supports for streaming compression.
+:::
+
+[apache-arrow-compression]: https://arrow.apache.org/docs/cpp/api/utilities.html#compression
+
+### `level = int (optional)`
+
+The compression level to use. The supported values depend on the codec used. If
+omitted, the default level for the codec is used.
```

Comment:
The range of valid values should be explicit now, per compression algorithm.

(This applies to all docs pages.)

Reply:
I don't think we can feasibly spell this out. For example, Zstd just a year ago or so added level -3. Depending on the version of libzstd used, the available levels can change.